sphinxgallery/gen_rst.py

```diff
@@ -445,7 +445,7 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf, plot_gallery):
     Returns the set of functions/classes imported in the example.
     """
     base_image_name = os.path.splitext(fname)[0]
-    image_fname = '%s_%%03d.png' % base_image_name
+    image_fname = 'sphx_glr_%s_%%03d.png' % base_image_name
```

Comment:
The right thing to do would be to implement what Gaël was saying in #5.

> How about we store a list of images generated by sphinx-gallery and delete only these. I suspect it's slightly harder than what I just said, but that's the spirit :)

If that's too hard, I guess we can live with this PR as a stop-gap solution/hack.

Stupid questions: 
* where is the code that copies the images from doc/auto_examples/images to _build/html/_images ? Is that inside sphinx or inside sphinx-gallery?
* where does the matplotlib plot directive stores its output images, is that _build/html/_images ?
* what are the proper variable names in sphinx-gallery for both of these folders?